build_system/cpack_install.cmake.in

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2006-2024, Knut Reinert & Freie Universität Berlin
+# SPDX-FileCopyrightText: 2016-2024, Knut Reinert & MPI für molekulare Genetik
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Only if creating the source package (`make package_source`):
+# Copy dependency include directories into package's staging folder
+if (CPACK_SOURCE_INSTALLED_DIRECTORIES)
+    file (COPY "@SEQAN3_SDSL_INCLUDE_DIR@/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/include/seqan3/vendor")
+    file (COPY "@SEQAN3_CEREAL_INCLUDE_DIR@/" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/include/seqan3/vendor")
+    configure_file ("@CPM_DOWNLOAD_LOCATION@" "${CMAKE_CURRENT_BINARY_DIR}/build_system/CPM.cmake" COPYONLY)
+    if ("@use_ccache_ADDED@" STREQUAL "YES")
+        configure_file ("@use_ccache_SOURCE_DIR@/ccache/CMakeLists.txt"
+                        "${CMAKE_CURRENT_BINARY_DIR}/test/cmake/seqan3_require_ccache.cmake" COPYONLY)
+    endif ()
+endif ()
```

Comment:
Random thoughts (no action required):
I wonder if sometime in the future, something like this could be replaced by some CPM package.